apps/member-profile/app/shared/components/company-review.tsx

```diff
@@ -31,8 +37,10 @@ type CompanyReviewProps = {
   reviewerId: string;
   reviewerProfilePicture: string | null;
   reviewedAt: string;
+  studentId?: string;
```

Comment:
We can change `studentId` to `userId` as mentioned earlier, but also we shouldn't need to make this value optional since there will always be a value corresponding to the current user session.